ecosystem/test/test.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 The Photon Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+
+
+int main(int argc, char **argv) {
```

Comment:
Seems this empty test caused SIGBUS in ARM.

What is the purpose of this empty test program?

Reply:
it actually links with test_simple.dom.cpp (and potentially other .cpp files in the future).

Reply:
Got it . Just like `gtest_main`.